lib/token_validator/token_service.rb

```diff
@@ -52,12 +52,12 @@ def valid_structure?
   def valid_scope?
     raise InvalidScope, 'Missing scopes' unless decoded_jwt.key?('scopes')
 
-    valid = true
+    valid = false
```

Comment:
Yikes, we should have always defaulted to `false`!

Reply:
Innocent until proven guilty?  :P

Reply:
I almost commented on this and then realized is the old code